packages/react-core/src/components/Menu/MenuInput.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Menu/menu';
+
+export interface MenuInputProps extends React.HTMLProps<HTMLElement> {
+  /** Items within input */
+  children?: React.ReactNode;
+  /** Forwarded ref */
+  innerRef?: React.Ref<any>;
+}
+
+const MenuInputBase: React.FunctionComponent<MenuInputProps> = ({ children }: MenuInputProps) => (
+  <div className={styles.menuSearch}>{children}</div>
+);
+
+export const MenuInput = React.forwardRef((props: MenuInputProps, ref: React.Ref<HTMLElement>) => (
+  <MenuInputBase {...props} innerRef={ref} />
+));
```

Comment:
Same as above, not really needed to add new component
```suggestion
export const MenuInput = React.forwardRef((props: MenuInputProps, ref: React.Ref<HTMLElement>) => (
  <div {...props} className={css(styles.menuSearch, props.className)} ref={ref} />
));
```